### DISCUSSION TITLE
The title in a MATE or Cinnamon menu contains "(color depth=16)"

### DISCUSSION THREAD

**Discussion by InfoLibre** [ImageMagick/ImageMagick#2739] [Development]
In a MATE or a Cinnamon menu (Linux Mint 20), the title of ImageMagick is strange, it contains **(color depth=16)**
Shouldn't it better to only write ImageMagick ?

![87197351-a390a680-c2fa-11ea-8f57-c4541c767aa3](https://user-images.githubusercontent.com/3955499/96367177-68e5a800-114c-11eb-8851-5e5c5e6e6af0.png)

![Capture d’écran du 2020-10-18 14-16-09](https://user-images.githubusercontent.com/3955499/96367191-874ba380-114c-11eb-8ea4-d2c0034a1b1b.png)

**Comment by dlemstra** [comment]
I don't think that we can controle how that is written in the menu. If you think we can them please let us know where/how we can change this.

**Comment by InfoLibre** [comment]
It's in the file /usr/share/applications/display-im6.q16.desktop
(package imagemagick-6.q16)

![Capture d’écran du 2020-10-23 20-51-12](https://user-images.githubusercontent.com/3955499/97042567-8910cf00-1571-11eb-8172-a4b259e9a81e.png)

**Reply by dlemstra** [reply]
That is not a file that we maintain so you will need to report that somewhere (no clue where) else.
